src/components/filmDetailsPage/Page.vue

```diff
@@ -2,17 +2,17 @@
   <div class="main-block">
     <div class="firs-block">
       <FilmInfo
-        imageSrc="https://www.vintagemovieposters.co.uk/wp-content/uploads/2020/12/IMG_1021-scaled.jpeg"
-        title="Back To The Future"
-        year="1985"
-        genre="Sci-Fi"
-        rating="8.8"
-        description="Back to the Future is a 1985 American science fiction film directed by Robert Zemeckis. Written by Zemeckis and Bob Gale, it stars Michael J. Fox, Christopher Lloyd, Lea Thompson, Crispin Glover, and Thomas F. Wilson. Set in 1985, the story follows Marty McFly (Fox), a teenager accidentally sent back to 1955 in a time-traveling DeLorean automobile built by his eccentric scientist friend Doctor Emmett 'Doc' Brown (Lloyd)"
-        duration="155"
+        :imageSrc="movie.poster_path"
```

Comment:
Consider using a single prop "movie" for FilmInfo component and then use destructuring with default values there.
Page.vue component should know about implementation details of FilmInfo and the data it uses.

Reply:
Fixed